2.6/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -151,8 +147,10 @@ function mongo_primary_member_addr() {
 
 # mongo_remove removes the current MongoDB from the cluster
 function mongo_remove() {
-  local primary_addr
-  primary_addr="$(mongo_primary_member_addr)"
+  local primary_addr="$(mongo_primary_member_addr ||  true)"
```

Comment:
Thanks, updated.

I can add any comment that you will like, because I don't know what to write here. After adding `|| true` code became self-readable for me and I wouldn't add comment here.